packages/notificator/notificator.go

```diff
@@ -54,8 +61,10 @@ func SendNotifications() {
 				return
 			}
 			id, _ := strconv.ParseInt(notif["id"], 10, 64)
-			if *notifications[ecosystemID].lastNotifID < id {
-				*notifications[ecosystemID].lastNotifID = id
+			if *notifications.storage[ecosystemID].lastNotifID < id {
+				notifications.Lock()
```

Comment:
I think, it would be better to move this as a Set method to ConcurrentNotifications structure